components/director/pkg/graphql/app_validation.go

```diff
@@ -0,0 +1,28 @@
+package graphql
+
+import (
+	validation "github.com/go-ozzo/ozzo-validation"
+	"github.com/go-ozzo/ozzo-validation/is"
+	"github.com/kyma-incubator/compass/components/director/pkg/inputvalidation"
+)
+
+func (i ApplicationCreateInput) Validate() error {
+	return validation.ValidateStruct(&i,
+		validation.Field(&i.Name, validation.Required, inputvalidation.Name),
+		validation.Field(&i.Description, validation.Length(0, 128)),
+		validation.Field(&i.Labels, inputvalidation.EachKey(validation.Required)),
+		validation.Field(&i.HealthCheckURL, validation.NilOrNotEmpty, is.URL, validation.Length(1, 256)),
```

Comment:
Why do we need to check the length and check whether the input is a valid URL?

Reply:
We should also allow URLs longer than 256.

Reply:
The URL has to be 256 long, because in database we have  `healthcheck_url varchar(256)`.